### DISCUSSION TITLE
Unable to Login to GitHub Enterprise

### DISCUSSION THREAD

**Discussion by jimmybrancaccio** [community/community#3237] [Enterprise] [answered]
Greetings,

With the announcement of GitHub Enterprise 3.0.0-rc1 I've been attempting to login to a test instance using the GitHub mobile app. As [noted here it says](https://docs.github.com/en/enterprise-server@3.0/github/getting-started-with-github/github-for-mobile) you should long press the Profile icon and then add the GitHub Enterprise account. Unfortunately doing a long press on the Profile icon does nothing. Any ideas?

iPad Pro w/ iOS 14.3
GitHub mobile for iOS
Version 1.4 (25456972)

Thanks,

**Comment by candyho** [comment, marked as answer]
Hey there, we're slowly rolling out the support of GitHub Enterprise 3.0 and it will be fully available for all iOS users in the next build that is shipping next Tuesday/Wednesday (version **1.4.2**). In the meantime, you can get Github Enterprise support through using our [iOS Beta app(version 1.4)](https://testflight.apple.com/join/NLskzwi5). Apologies for the misleading copy in the help doc and the inconvenience caused!

**Reply by jimmybrancaccio** [reply]
Gotcha! Thanks @candyho!
